src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -2340,6 +2359,20 @@ internal static string ConcatenateStringPathArguments(CommandElementAst stringAs
                         break;
                     }
 
+                case "Export-Alias":
```

Comment:
In C#, empty `case` blocks fall through to the next non-empty `case`.  So if you reached the "Export-Alias" case since there's no code there it'll try to skip through to the next case "Import-Alias" (which is empty" so skip to next one and next one until you reach a non-empty case "Set-Alias"..so the question is do you want the empty cases (Export-Alias, Import-Alias, New-Alias, New-Variable, Remove-Alias) to all reach the "Set-Alias" case and execute the logic there? If so, your code is okay. If that's not the expected behavior add the line `break;` to each currently empty case. 
```suggestion
                case "Export-Alias":
                    break;
```

Reply:
Yep! Each of these cmdlets use the scope parameter, so the fall through is intended